example/todoDbWithLoom/app/src/TodoMvcDbWithLoom.scala

```diff
@@ -1,40 +1,31 @@
 package app
-import scalasql.DbApi.Txn
```

Comment:
Is there a reason you are removing this import? Seems like it makes all the code in the file a lot more verbose needing to spell out `scalasql.core.DbApi.Txn` over and over